### DISCUSSION TITLE
Error: "instanceInfo" is undefined after upgrading to 7

### DISCUSSION THREAD

**Discussion by dword-design** [typegoose/mongodb-memory-server#497] [Q&A] [answered]
## Versions

- NodeJS: 0.0.0
- mongodb-memory-server-*: 7.0.0
- mongodb(the binary version): 14.17.0

package: mongo-memory-server 

## What is the Problem?

After upgrading to version 7, my tests aren't running anymore. Instead, I'm getting the above error. Usage is simply creating a server and querying the URI.

## Code Example

```ts
const mongo = new MongoMemoryServer()

const localUrl = await mongo.getUri()
```

## Do you know *why* it happenes?

Looks like the server isn't started via the constructor, as opposed to the readme.

**Comment by hasezoey** [comment, marked as answer]
- https://nodkz.github.io/mongodb-memory-server/docs/guides/migrate7#no-function-other-than-start-create-ensureinstance-will-be-starting-anything
- https://nodkz.github.io/mongodb-memory-server/docs/guides/migrate7#geturi-is-no-longer-async

PS: this is basically a duplicate of https://github.com/nodkz/mongodb-memory-server/discussions/458

**Reply by dword-design** [reply]
Great thanks, it's working.
